openff/toolkit/utils/nagl_wrapper.py

```diff
@@ -17,6 +17,12 @@
 
 
 class NAGLToolkitWrapper(ToolkitWrapper):
+    """NAGL toolkit wrapper for applying partial charges with a GCN model.
```

Comment:
I'm marginally concerned that the prose uses "NAGL" and "OpenFF NAGL" synonymously. I don't think now is the time to advocate for changing the name of the class ... but it'd be nice to consistently use "OpenFF NAGL" in text where practical